src/test/java/parser/ParserTest.java

```diff
@@ -0,0 +1,43 @@
+package parser;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I think explicit imports would better adhere to the coding standard. Maybe this is an unintentional mistake caused by the IDE?